src/app/settings/views/Configuration/GeneralForm/GeneralForm.tsx

```diff
@@ -120,8 +120,9 @@ const GeneralForm = (): JSX.Element => {
             Use MAAS name and unicode emoji(s) to describe your MAAS instance.{" "}
             <br />
             <br />
-            Examples: <br />⛔ maas-prod <br />
-            my-maas ⚠️ no-deploys
+            Examples: <br />
+            US-west-2🇺🇸MAAS-prod <br />
+            my-maas❗no-deploys
```

Comment:
Missing space (according to [original suggestion](https://github.com/canonical/maas-ui/issues/4785#issuecomment-1446201400))
```suggestion
            US-west-2 🇺🇸 MAAS-prod <br />
            my-maas ❗ no-deploys
```

Reply:
Damn, I forgot that we *can* have spaces for MAAS names!